storage/innobase/include/sync0policy.h

```diff
@@ -137,8 +149,9 @@ class MutexDebug {
 		/** Line mumber in filename */
 		unsigned	m_line;
 
+	private:
 		/** Thread ID of the thread that own(ed) the mutex */
-		os_thread_id_t	m_thread_id;
+		ulint		m_thread_id;
```

Comment:
Since you changed type to ```ulint```, it would be nice to use ```my_atomic_loadlint()``` / ```my_atomic_storelint()```.

Reply:
Thanks for noticing that.